cfgmgr/buffermgr.cpp

```diff
@@ -399,6 +453,12 @@ void BufferMgr::doTask(Consumer &consumer)
         return;
     }
 
+    if (table_name == CFG_PORT_QOS_MAP_TABLE_NAME)
+    {
+        doPortQosTableTask(consumer);
+        return;
```

Comment:
A rare scenario is that the `pfc_enable` of a port is received after the `speed` of any port. In that case, the flow returns here and the lossless PG will not be generated. Maybe call `doSpeedUpdateTask` from `doPortQosTableTask`?

Reply:
Fixed. Could you help review again? Thanks